### PR TITLE
Import rework

### PR DESCRIPTION
**Pull request by norkans7**

### CODE REVIEW

**Comment by norkans7** [comment]
new_import_customize

**Comment by ericnewcomer** [comment]
Those select2 widgets on the left are pretty weird. What's wrong with pre-checked checkboxes there?

**Comment by norkans7** [comment]
I think we talked about having the user explicitly choose for every field whether to include or ignore

**Comment by nicpottier** [comment]
Ya, we had definitely talked about forcing an action there, but I think defaulting to selecting them (with a checkbox) is probably just as effective and a bit cleaner. (I think this is where we ended up on the call)

**Comment by norkans7** [comment]
Okay I revert to checkboxes

**Comment by norkans7** [comment]
font-checkboxes-true-default

**Comment by ericnewcomer** [comment]
Checkboxes look a whole lot better

File: temba/contacts/tests.py [hunk 1]
```diff
@@ -1816,6 +1825,15 @@ def test_contact_import(self):
...
 
+        post_data['column_joined_label'] = 'District'
+
+        response = self.client.post(customize_url, post_data, follow=True)
+        self.assertFormError(response, 'form', None, 'District should be used once')
+
+        post_data['column_joined_label'] = '[_NEW_]District'
+
+        response = self.client.post(customize_url, post_data, follow=True)
+        self.assertFormError(response, 'form', None, 'District should be used once')
```
Comment (nicpottier) [inline temba/contacts/tests.py]: Can we add a test that the user doesn't set two import fields to the same contact field?

**Comment by nicpottier** [comment]
Looks good, what about better error messaging when we reject records. (which records and why.. maybe we just do first 100..)

File: temba/contacts/tests.py (hunk 1 above)
Comment (norkans7) [inline temba/contacts/tests.py]: The tests above are just for that

**Comment by norkans7** [comment]
Depends on https://github.com/nyaruka/smartmin/pull/65

To Show why we reject a row when importing

better-error-description

**Comment by nicpottier** [comment]
Can we fix the build so we can push this?

**Comment by norkans7** [comment]
Ready, merged smartmin

**Pull request merged**